scripts/run_tests.sh

```diff
@@ -23,8 +23,14 @@ function display_result {
 make test-requirements
 display_result $? 1 "Requirements check"
 
+black ./app ./tests --check
+display_result $? 2 "Code style check (Black)"
+
 flake8 .
-display_result $? 1 "Code style check"
+display_result $? 1 "Code style check (flake8)"
+
+isort --check-only ./app ./tests
```

Comment:
Changed to `isort --check-only .` after to leverage the config file

Reply:
interesting - I guess we never had an isort check on this repo?

Reply:
Yeah, `isort` is an addition to the API!